druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -503,15 +493,25 @@ public Void apply(ResultSet input) {
             "gender=F; state_province=OR",
             "gender=M; state_province=WA",
             "gender=F; state_province=WA")
+        .queryContains(
+            druidChecker("{'queryType':'groupBy','dataSource':'foodmart',"
+            + "'granularity':'all','dimensions':[{'type':'default',"
+            + "'dimension':'gender'},{'type':'default',"
+            + "'dimension':'state_province'}],'limitSpec':{'type':'default',"
+            + "'columns':[{'dimension':'state_province','direction':'ascending'},"
+            + "{'dimension':'gender','direction':'descending'}]},"
+            + "'aggregations':[{'type':'longSum','name':'dummy_agg',"
+            + "'fieldName':'dummy_agg'}],"
+            + "'intervals':['1900-01-09T00:00:00.000/2992-01-10T00:00:00.000']}"))
         .explainContains(explain);
   }
 
   @Test public void testSortLimit() {
-    // Note: We do not push down SORT-LIMIT into Druid "groupBy" query yet
-    final String explain = "PLAN="
-        + "EnumerableInterpreter\n"
-        + "  BindableSort(sort0=[$1], sort1=[$0], dir0=[ASC], dir1=[DESC], offset=[2], fetch=[3])\n"
-        + "    DruidQuery(table=[[foodmart, foodmart]], intervals=[[1900-01-09T00:00:00.000/2992-01-10T00:00:00.000]], projects=[[$39, $30]], groups=[{0, 1}], aggs=[[]])";
+    final String explain = "PLAN=EnumerableLimit(offset=[2], fetch=[3])\n"
```

Comment:
i guess this can be tricky, since druid does not support offsets -> i need to alter the limit and return let say 5 instead of 3 and let calcite do the offset ?